module/mempool/stdmap/queues.go

```diff
@@ -53,7 +54,11 @@ func (b *Queues) Get(queueID flow.Identifier) (*queue.Queue, bool) {
 }
 
 func (b *Queues) Run(f func(backdata *QueuesBackdata) error) error {
+	p1 := binstat.EnterTime("~4lock:w:Backend.Run(Queues)", "")
```

Comment:
**A general comment on this PR**: all these strings passed to `EnterTime` are recommended to be replaced by `const` type.

Reply:
@yhassanzadeh13 thanks for the comment! There are many strings like this which are slightly different, and used only once. Not sure it makes sense to turn them into `const` type strings. It is true that the base of the strings can be used multiple times. Maybe these could be made in `const` type strings and then appended with non-`const` type string modifiers? All in all, I'm not sure if this idea makes sense but I'm open to discussing it further! :-)

Reply:
@simonhf thanks for your reply, I think having them either on `const` or using string modifiers is still safer than inline strings like what we have here.